### ISSUE TITLE
Removing trailing/leading spaces in directives and bindings

### ISSUE THREAD

**Issue by Ericlm** [vuejs/eslint-plugin-vue#2482]
**Please describe what the rule should do:**
The rule would remove trailing and leading spaces from within directives and dynamic attributes. It could also remove trailing/leading spaces from static attributes maybe?

**What category should the rule belong to?**
[X] Enforces code style (layout)
[ ] Warns about a potential error (problem)
[ ] Suggests an alternate way of doing something (suggestion)
[ ] Other (please specify:)

**Provide 2-3 code examples that this rule should warn about:**

```vue
<span v-if=" condition " /> ⚠️ Unexpected trailing and leading spaces

<span
    v-for=" i of 10 " ⚠️ Same here
    :key="i " ⚠️ Warns about trailing space
/>

<span
    class=" bg-red-100 text-blue-900 " // ⚠️ Trailing & leading
/>
```

**Comment by FloEdelmann** [comment]
That's rather formatter territory than linter territory. But we currently do have many formatter-like rules, so this could be considered. @ota-meshi what do you think?

**Comment by ota-meshi** [comment]
In my opinion, it would be useful to provide formatting rules in an ESLint plugin, since current formatters, like Prettier, don't accept options in most cases.

**Comment by Ericlm** [comment]
Maybe it could be integrated into eslint-stylistic ? There are already support for ts,js,jsx ; maybe would it be possible to have those rule into this package ? That would also allow the rules to be automatically "inherited" from the stylistic config.
